http-observatory/s3.py

```diff
@@ -0,0 +1,28 @@
+import boto
+import boto.s3
+import os
+import sys
+
+
+def send_to_s3(file_path):
+    """
+    Upload a file to a given s3 bucket
+    """
+    file = open(file_path)
```

Comment:
Don't overwrite built-in python functions, such as `file()`. Also, if you're using open, it's probably best to use `with`.